package.json

```diff
@@ -95,5 +62,29 @@
       "path": "./node_modules/cz-conventional-changelog"
     }
   },
-  "version": "0.0.0-development"
+  "jest": {
+    "testEnvironment": "node",
+    "setupFiles": [
+      "./testSetup.js"
+    ]
+  },
+  "greenkeeper": {
+    "ignore": [
+      "cosmiconfig"
+    ]
+  },
+  "version": "0.0.0-development",
+  "keywords": [
+    "lint",
+    "git",
+    "staged",
+    "javascript",
+    "css",
```

Comment:
We should probably remove irrelevant keywords 😳

Let's remove `css`, `sass`, `scss`, `javascript`?

Reply:
And add `prettier` and `formatting`

Reply:
Or `format`?

Reply:
oh yeah, `keywords` was out of my sight 😆 
> Let's remove css, sass, scss, javascript?

👍 

> And add prettier and formatting
> Or `format`?

`check`, `format`, `validate`?

Reply:
Good additions. Please add them all

Reply:
done 👍